native/kotlin/api/kotlin/src/integrationTest/kotlin/MediaEndpointTest.kt

```diff
@@ -72,7 +72,7 @@ class MediaEndpointTest {
             requestBuilder.media().create(
                 params = MediaCreateParams(title = title),
                 "test_media.jpg",
-                "image/jpg"
+                "image/jpeg"
```

Comment:
It's a little concerning that `image/jpg` no longer works in WordPress `6.8`. It might be worth looking into implementing a `SupportedFileContentType` `enum` to make it easier for clients to set this field.

cc @jkmassel

Reply:
Is this part of the release notes, or just observed behaviour?

This feels more like a server thing...

Reply:
It was observed behaviour. Without this change, create media test is failing with:

> WpError(errorCode=uniffi.wp_api.WpErrorCode$CustomException: v1=rest_upload_image_type_not_supported, errorMessage=The web server cannot generate responsive image sizes for this image. Convert it to JPEG or PNG before uploading., statusCode=400, response={"code":"rest_upload_image_type_not_supported","message":"The web server cannot generate responsive image sizes for this image. Convert it to JPEG or PNG before uploading.","data":{"status":400}})

I believe it's a new `WpError` type, because we don't have an enum variant for it yet.

Reply:
It's probably related to [this change](https://make.wordpress.org/core/2025/03/25/miscellaneous-developer-changes-in-wordpress-6-8/#media-control-uploading-of-unsupported-media-typesin-6-8-a-new-filter-wp-prevent-unsupported-mime-type-uploads-controls-the-behavior-of-the-editor-and-media-library-when-users-upload-an-image-type-that-the-serve-does-not-support-by-default-users-will-see-an-error-message-that-this-image-cannot-be-processed-by-the-web-server-convert-it-to-jpeg-or-png-before-uploading-developers-can-return-false-from-the-filter-to-enable-uploading-of-these-images-however-sub-sized-images-will-not-be-created-61167).

Reply:
Ah yep – [looks like a bug](https://make.wordpress.org/core/2025/03/25/miscellaneous-developer-changes-in-wordpress-6-8/#comment-47628)

I'm fine to keep our tests passing with this fix though